doc/man3/OSSL_PARAM_get_int32_t.pod

```diff
@@ -0,0 +1,148 @@
+=pod
+
+=head1 NAME
+
+OSSL_PARAM_get_TYPE, OSSL_PARAM_set_TYPE, OSSL_PARAM_get_BN, OSSL_PARAM_set_BN
+- OSSL_PARAM helpers
+
+=head1 SYNOPSIS
+
+=for comment generic
+
+ #include <openssl/params.h>
+
+ OSSL_PARAM_TYPE(key, address)
+
+ OSSL_PARAM_SIZED_TYPE(key, address, return_size)
+
+ OSSL_PARAM_SIZED_BN(key, address, size, return_size)
+
+ OSSL_PARAM_END
```

Comment:
I would remove the empty lines between these for intializer macros

Reply:
Done.